cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -159,13 +163,14 @@ protected boolean removeEldestEntry(Map.Entry<UUID, UserTaskInfo> eldest) {
    *                 returns a completed Future.
    * @param step The index of the step that has to be added or fetched.
    * @param isAsyncRequest Indicate whether the task is async or sync.
+   * @param parameters Help {@link UserTaskInfo} determine the response to persist.
    * @return The list of {@link OperationFuture} for the linked UserTask.
    */
   public List<OperationFuture> getOrCreateUserTask(HttpServletRequest httpServletRequest,
                                                    HttpServletResponse httpServletResponse,
-                                                   Function<String, OperationFuture> function,
-                                                   int step,
-                                                   boolean isAsyncRequest) {
+                                                   Function<String, OperationFuture> function, int step,
```

Comment:
Why do we put `step` to the same line as `function` -- is this our coding convention?